content/blog/2021/02/2021-02-08-docker-base-os-upgrade.adoc

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Docker image updates"
+tags:
+- announcement
+- docker
+- platform-sig
+author: markewaite
+opengraph:
+  image: /images/docker/dockerJenkins-updated-os-2021-02.png
+---
+
+Beginning with Jenkins 2.279 and Jenkins 2.263.4, the Jenkins project is upgrading the base operating system and Java version used in the `jenkins/jenkins:latest` and `jenkins/jenkins:lts` images.
+The update includes the most recent AdoptOpenJDK 8u282 release and Debian 10 (link:https://www.debian.org/releases/buster/["Buster"]).
+The Java update includes fixes made in JDK 8u252, JDK 8u262, JDK 8u272, JDK 8u275, and JDK 8u282.
+
+Other Docker images are already using more current operating system releases and Java versions.
+
+image:/images/docker/dockerJenkins.png[Jenkins and Docker, role=center, float=right, height=224]
+
+== Packaging changes
+
+The Jenkins Docker image based on Debian 10 ("Buster") includes some different packages than Debian 9 ("Stretch").
+Some packages have been removed because they are no longer supported by their communities.
+Some packages have been removed due to infrequent and decreasing use.
+Users of the Jenkins Docker images may need to extend their definition of their Docker image to include packages that are no longer included in the base image.
+
+=== SCM packages removed
+
+The following software configuration management packages are no longer included in the Jenkins controller images for `jenkins/jenkins:latest` or for `jenkins/jenkins:lts`:
+
+* bzr
+* mercurial
+* subversion
+
+=== Other packages removed
+
+Additional packages that are no longer included in the Jenkins controller images include:
+
+* bzip2
+* mime-support
+* python (the Python project stopped supporting Python 2 January 1, 2020)
+* xz-utils
+
+A detailed list of the exact package changes is available in the link:https://github.com/jenkinsci/docker/pull/1070[pull request].
+
+See the <<upgrade guidelines,upgrade-and-compatibility-notes>> below for Dockerfile customizations that include those packages.
+
+== Why?
+
+We're changing the base image so that we have a better supported operating system and a more current Java release for Jenkins controllers.
+
+=== Better supported operating system
+
+The Docker images provided by the Jenkins project rely on the operating system security processes of the operating system provider.
+
+Our Docker images have used Debian 9 (link:https://www.debian.org/releases/stretch/["Stretch"])) for multiple years.
+Debian 9 security updates have been link:https://www.debian.org/releases/stretch/[discontinued as of July 6, 2020].
+Debian 9 Long Term Support security updates will be discontinued at the link:https://www.debian.org/releases/stretch/[end of June 2022].
```

Comment:
Move to the top of the post?

Reply:
Moved to the top in cf9b6bbf1f94e571d005e5d65783ac763677793d